### PR TITLE
service account for QE Jenkins

### PR DESCRIPTION
**Pull request by akostadinov**
https://jira.coreos.com/browse/DPTP-303

### CODE REVIEW

**Comment by petr-muller** [comment]
/test pj-rehearse-testing

(just testing a potential new `openshift/release` presubmit, it's supposed to pass here)

**Comment by stevekuznetsov** [comment]
/lgtm
/hold
@smarterclayton any comments

File: cluster/ci/config/roles.yaml [hunk 1]
```diff
@@ -22,6 +22,12 @@ items:
   metadata:
     name: telemeter
 
+- kind: ServiceAccount
+  apiVersion: v1
+  metadata:
+    name: qe-jenkins
+    namespace: ocp
```
Comment (smarterclayton) [inline cluster/ci/config/roles.yaml]: It doesn't need to be in the ocp namespace.  Any service account token on the cluster can access these, so create a QE specific project and we'll assign it there.

File: cluster/ci/config/roles.yaml [hunk 2]
```diff
@@ -274,3 +280,16 @@ items:
     kind: ClusterRole
     apiGroup: rbac.authorization.k8s.io
     name: system:image-puller
+- kind: RoleBinding
```
Comment (smarterclayton) [inline cluster/ci/config/roles.yaml]: Unnecessary, all accounts on the cluster can pull these secrets.

Comment (akostadinov) [reply cluster/ci/config/roles.yaml]: The goal is to be able to pull images from the ocp namespace like 

> registry.svc.ci.openshift.org/ocp/release:4.0.0-0.nightly-2019-02-08-145216

@smarterclayton , are you saying that RoleBinding can be removed and this service account will still be able to pull images from `ocp` namespace? Not sure what secrets you mean.

Comment (akostadinov) [reply cluster/ci/config/roles.yaml]: Ok, I removed this RoleBinding. Thank you.

**Comment by smarterclayton** [comment]
Create your own namespace.  You can use the default service account.  You won't need any role bindings except granting access to the person who needs to read the service account token.

**Comment by stevekuznetsov** [comment]
> You won't need any role bindings except granting access to the person who needs to read the service account token.

You need `edit` to run `oc serviceaccounts get-token` FYI, bind these roles to users by _GitHub_ username

**Comment by akostadinov** [comment]
@stevekuznetsov , would you help me figure out why `ci/prow/generated-config` failed? I don't think I changed anything related to it?

**Comment by akostadinov** [comment]
@stevekuznetsov , you want me to change rolebinding to `edit` from `admin`?
@smarterclayton , can I keep using `cci-jenkins` service account to make clear where that is used?

**Comment by petr-muller** [comment]
@akostadinov the `generated-config` test was failing because we had the repo in an inconsistent state for a while. Not your fault.

**Comment by stevekuznetsov** [comment]
Your current approach also LGTM
@smarterclayton PTAL
/hold
/lgtm

**Pull request merged**